packages/api-docs-builder/utils/getPropsFromComponentSymbol.ts

```diff
@@ -0,0 +1,236 @@
+import * as ts from 'typescript';
```

Comment:
This file is adapter from `typescript-to-proptypes`
My end-goal is to refacto `typescript-to-proptypes` to use the same parsing logic and just have a different output format (the `checkSymbol` part).